packages/block-library/src/site-logo/editor.scss

```diff
@@ -28,6 +24,10 @@
 		height: auto;
 		max-width: 100%;
 	}
+
+	.wp-block-image__crop-area {
+		pointer-events: auto;
+	}
```

Comment:
Note: As a follow-up, we'll want to add this style to the Image block as well. The current Image block does not accept mouse operations when the Crop tool is enabled.